### PR TITLE
Setting up bootstrap

### PR DESCRIPTION
**Pull request by holeyness**

### CODE REVIEW

**Comment by nicholasyan** [comment]
Looks good to me.

File: app/tsconfig.json [hunk 1]
```diff
@@ -10,10 +10,11 @@
...
    "files": [
-	"./src/ts/app.ts",
-    "./src/ts/upload.ts",
-    "./src/ts/play.ts",
-	"./src/ts/video.ts"
+	"src/ts/app.ts",
+    "src/ts/upload.ts",
+    "src/ts/play.ts",
+	"src/ts/video.ts",
+     "src/ts/bootstrap.js"
```
Comment (ethanpailes) [inline app/tsconfig.json]: js files should not be part of the list of files for the typescript compiler to build. Please remove this line.

**Comment by ethanpailes** [comment]
Is it possible to configure bootstrap easily with npm, or is the state of the art basically just copying the files?

**Review by ethanpailes — CHANGES_REQUESTED** [review]

**Comment by holeyness** [comment]
Two things: 
1. bootstrap can be packaged with npm, but then it's installed under like node_modules, so the importing of css and scripts file is more of a pain, not sure what the correct solution for that is.
2. As for the js file, if we use the copy paste method, like whats the pattern there? Cuz nothing in the js folder is committed
@ethanpailes

**Comment by ethanpailes** [comment]
So it sounds like the plan is just to put it in the js directory and ungitignore it.

**Pull request merged**